### DISCUSSION TITLE
Playlist Tracks limited to 100 on YouTube Music

### DISCUSSION THREAD

**Discussion by AnthonyJWinslow** [music-assistant/support#1110] [Q&A] [answered]
Is this a feature or a bug, my YouTube Music playlists are only loading the first 100 tracks? I have a playlist with 238 tracks but I can only seem to view and play 100 of them from MusicAssistant. Is there a setting / way to load all the tracks?

Yes, I have googled this, no I cannot find anything similar or a fix...

[EDIT] I did verify it only has 100 tracks to verify it was not the same bug that was fixed in 2022.7.2 where it was just showing the incorrect number.

![image](https://user-images.githubusercontent.com/93484039/221920184-c1e68292-97f1-48f2-b4d8-a0c7cc26830d.png)

**Comment by OzGav** [comment, marked as answer]
Please also review the open issues as well as googling
https://github.com/music-assistant/hass-music-assistant/issues/821
https://github.com/music-assistant/hass-music-assistant/issues/1072

**Reply by AnthonyJWinslow** [reply]
1072 is exactly my issue! IDK how I missed that thread. Thank your for finding these.
